
Turn 4
================================================================
error: failed to decode bot message: "T" (at position 1)
Last input received was:
Traceback (most recent call last):
  File "MyBot.py", line 164, in <module>
    ships.sort(key= sortByPriority)
  File "MyBot.py", line 133, in sortByPriority
    return item not in ships_collecting
  File "D:\Python-Workspace\Halite\Trabalho-IA\choi_yena\hlt\entity.py", line 36, in __eq__
    return self.owner == other.owner and self.id == other.id
AttributeError: 'int' object has no attribute 'owner'

